4.0/asvs.py

```diff
@@ -203,3 +215,18 @@ def to_csv(self):
         writer.writerows(self.asvs_flat)
 
         return si.getvalue()
+
+    def to_opencontrol(self):
+        ''' Returns opencontrol yaml '''
+
+        opencontrol_result=yaml.dump({'name':f"{self.asvs['ShortName']}-{self.asvs['Version']}"})
+        for key in self.sorted_nicely(self.asvs_opencontrol.keys()):
+            opencontrol_result+=yaml.dump({
+                key:{
+                    "family":self.asvs_opencontrol[key]["family"],
+                    "name":self.asvs_opencontrol[key]["name"],
+                    "description":self.asvs_opencontrol[key]["description"]
```

Comment:
Not in the standard but there is a separate schema for certifications where you can define which of the levels requires which controls.
https://github.com/opencontrol/schemas/blob/master/kwalify/certification/v1.0.0.yaml

Here is an example how to generate them:
```bash
for level in 1 2 3;  do 
  FILENAME=owasp-asvs-4x-L${level}.yaml; 
  cat << EOH > ${FILENAME}
name: OWASP ASVS 4.x Level ${level}
standards:
  ASVS-4.x:
EOH
  cat OWASP\ Application\ Security\ Verification\ Standard\ 4.0-en.flat.json | \
    jq -r ".requirements[]|select(.L${level}==\"X\")|\"    \(.Item): {}\"" | sort -V >> ${FILENAME}
done
```

Reply:
@yruss972 Would you like to add this output format as well?

Reply:
I'll try to polish it up so it will handle versioning and add it to the PR.
What would you think about setting up the repo to be used directly as an opencontrol dependency, certifications and all?

Reply:
It sounds great! Do you know how people would verify against it? Do you have any documentation on what this involves?

Reply:
Has this thread been resolved?

Reply:
Ike I do not think so, can we preserve this work and/or can we close this PR?